src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menAge = people
+    .filter((man) => arguments.length < 2
+      ? man.sex === 'm'
+      : man.sex === 'm' && Math.ceil(man.died / 100) === century)
+    .map((index) => index.died - index.born);
```

Comment:
Consider using a more descriptive name for the variable `index` in the `.map()` function. It represents a person, so you could rename it to `person`.